fiaas_deploy_daemon/usage_reporting/dev_hose_auth.py

```diff
@@ -29,13 +29,13 @@
 class DevHoseAuth(AuthBase):
     def __init__(self, key, tenant):
         self._key = base64.b64decode(key.strip())
-        self._auth_context = base64.b64encode(json.dumps({"type": tenant}).encode("utf-8")).decode("utf-8")
```

Comment:
Thanks for the python3 related fixes here 👍. If I'm reading this correctly, the functional change here is that `_auth_context` is bytes when it is passed to `_create_string_to_sign` below. Since the tests for this passed in the python3 changeset, is there a way to change the tests so that they also verify this behavior?

Reply:
I do not find how to change the test to verify this behaviour. I did the other changes but I'm not able to do this one

Reply:
That's okay, thanks for checking